formats/MrssFormat.php

```diff
@@ -143,6 +144,12 @@ public function stringify(){
 				$entry->appendChild($entryDescription);
 			}
 
+			if (!empty($itemDuration)) {
```

Comment:
With strict typing, explicit comparison might be preferred.

```suggestion
			if ($itemDuration !== null) {
```